css-view-transitions-2/Overview.bs

```diff
@@ -753,6 +753,72 @@ div.box {
 		1. Return |viewTransition|.
 	</div>
 
+# Determining 'view-transition-name' automatically # {#auto-vt-name}
+
+## Overview ## {#auto-vt-name-overview}
+
+*This section is non-normative.*
+
+For an element to participate in a view transition, it needs to receive a unique 'view-transition-name'.
+This can be tedious and verbose when multiple elements are involved in the same view transition, especially in cases where many of those are same-element transitions,
+as in, the element has the same 'view-transition-name' in the old and new state.
+
+To make this easier, setting the 'view-transition-name' to ''view-transition-name/auto'' would generate a 'view-transition-name' for the element, or take it from the element's [=Element/id=] if present.
+
+## Examples ## {#auto-vt-name-example}
+
+<div class="example">
+In this example, view transition is used to sort a list in an animated way:
+
+```html
+<ul>
+	<li>Item 1</li>
+	<li>Item 2</li>
+	<li>Item 3</li>
+	<li>Item 4</li>
+	...
+</ul>
+```
+
+Ordinarily, each of these items would have to receive a unique 'view-transition-name':
+
+```css
+li:nth-child(1) { view-transition-name: item1; }
+li:nth-child(2) { view-transition-name: item1; }
+li:nth-child(3) { view-transition-name: item1; }
+li:nth-child(4) { view-transition-name: item1; }
+...
+```
+
+With ''view-transition-name/auto'', this CSS would work:
+
+```css
+li {
+	view-transition-name: auto;
+}
+```
+</div>
+
+## Additions to 'view-transition-name' ## {#additions-to-vt-name}
+
+In addition to the existing values, the 'view-transition-name' also accepts an <dfn for=view-transition-name type=value export>auto</dfn> keyword.
```

Comment:
It would have been useful for w3c tools, to define it with `New values` in a (partial) property definition table ([exemple](https://github.com/w3c/csswg-drafts/blob/0ab99f38873352f272ec0f430c10084f9621bbe7/css-anchor-position-1/Overview.bs#L805)).